### PR TITLE
fix(web): reset search input after switching models or views

### PR DESCRIPTION
**Pull request by caichi-t**
# Overview
This PR is fixing the issue of the search input remaining after switching models or views.
And, fixing page to go back to 1 after switching views.

## What I've done
- Removed page, pageSize and searchTerm from URL params
- Input values and page are reset after switching models or views
- Page is reset after adding or removing filter
- Added id property to currentView.

### CODE REVIEW

File: web/src/components/molecules/Content/Table/index.tsx [hunk 1]
```diff
@@ -495,14 +500,14 @@ const ContentTable: React.FC<Props> = ({
...
           placeholder={t("Please enter")}
-          defaultValue={searchTerm}
           onSearch={(value: string) => {
             if (value) {
               onSearchTerm(value);
             } else {
               onSearchTerm();
             }
           }}
+          key={`${modelKey}${currentView.id}`}
```
[the remark this answers is not in the document]
Comment (nourbalaha) [reply web/src/components/molecules/Content/Table/index.tsx:510]: You can do `value={searchTerm}` and update a handleChange method.

[the remark this answers is not in the document]
Comment (caichi-t) [reply web/src/components/molecules/Content/Table/index.tsx:510]: In that way, the input value isn't reset when switching models or views while only inputting value and not searching.
I think it's confusing that the value remains.

[the remark this answers is not in the document]
Comment (caichi-t) [reply web/src/components/molecules/Content/Table/index.tsx:510]: If we search every time we change one letter, that way would be good.

[the remark this answers is not in the document]
Comment (nourbalaha) [reply web/src/components/molecules/Content/Table/index.tsx:510]: if you do my suggestion you can setSearchTerm to empty string when changing the view.

**Review by nourbalaha — APPROVED** [review]

**Pull request merged**